tdrs-backend/docs/api/roles.md

```diff
@@ -0,0 +1,83 @@
+
+# Roles 
+Accepts GET and Post requests from [authenticated](api/authentication.md) Admin users to create and update  roles. 
```

Comment:
This doc looks out of date to me -- my understanding is that admin users are no longer able to create/update roles via the API since that will now be done via the Django Admin dashboard.

Reply:
That's correct. I missed that the `POST` endpoint is still mentioned here.

Reply:
@alexsoble This is now updated accordingly

Reply:
Thanks! Though I'm still seeing "to create and update  roles" when I look at the changed files, should we remove that as well?

I'm also seeing "Accepts GET and requests".

Reply:
Yep, it's all set now

Reply:
Looks good!